src/WebJobs.Script.WebHost/ContainerManagement/LinuxContainerInitializationHostService.cs

```diff
@@ -55,6 +55,11 @@ private async Task ApplyStartContextIfPresent()
 
                 bool success = _instanceManager.StartAssignment(assignmentContext);
                 _logger.LogInformation($"StartAssignment invoked (Success={success})");
+
+                var error = await _instanceManager.SpecializeMSISidecar(assignmentContext);
```

Comment:
On a regular Assign, we specialize MSI first, then StartAssignment, but here the order is reversed. Does that matter?

Reply:
Order doesn't matter. In case the re-specialization of side car fails for some reason, dont want that impacting the re-specialization of main container. So switched the ordering around since the respecialization code in sidecar is not still well tested. After a couple releases we can make it consistent.